### ISSUE TITLE
Bug: EdgeFeatureHubConfig incorrectly removing "/" from host URL

### ISSUE THREAD

**Issue by icebirdmail** [featurehub-io/featurehub#411]
**Describe the bug**
When using the EdgeFeatureHubConfig constructor with a URL that ends in "/features/", the URL will be rewritten to remove the forward slash before features. E.g. "https://feature.featurehub.io/features/" becomes "https://feature.featurehub.iofeatures/"

 Which area does this issue belong to?

* [ ] FeatureHub Admin Web app
* [x] SDK
* [ ] SDK examples
* [ ] Documentation
* [ ] Other

**To Reproduce**
1. Follow the example in the documentation on how to set up a connection to FeatureHub using a host URL that ends in "/features/".
2. Observer the error and the URL that FeatureHub is trying to connect to.

**Expected behavior**
FeatureHub connects to the intended URL.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Windows
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] featurehub-javascript-client-sdk 1.0.2

You can get the version of the FeatureHub container by running ```docker ps``` command.
Please include the OS and what version of the OS and Docker you're running.

**Additional context**
The issue appears to relate to lines 29-45 of edge_featurehub_config.ts

**Comment by rvowles** [comment]
whoops!

**Comment by rvowles** [comment]
added a test: 

```
  it('should construct urls properly', () => {
    const fc = new EdgeFeatureHubConfig('http://localhost:8080', '123*345');
    expect(fc.getHost()).to.eq('http://localhost:8080/');
    expect(fc.url()).to.eq('http://localhost:8080/features/123*345');
  });

  it('should strip off /feature/ if a user provided it', () => {
    const fc = new EdgeFeatureHubConfig('https://feature.featurehub.io/features/', '123*345');
    expect(fc.getHost()).to.eq('https://feature.featurehub.io/');
    expect(fc.url()).to.eq('https://feature.featurehub.io/features/123*345');
  });
```

**Comment by rvowles** [comment]
Did you want to check the PR - I can have it merged and release a new version @icebirdmail ?

**Comment by IrinaSouth** [comment]
Released in version 1.0.3 https://www.npmjs.com/package/featurehub-javascript-client-sdk
and https://www.npmjs.com/package/featurehub-javascript-node-sdk
